webapp/dynamic_components_handler.go

```diff
@@ -41,5 +41,7 @@ func flagsComponentHandler(w http.ResponseWriter, r *http.Request) {
 		log.Errorf("Error loading flags: %s", err.Error())
 	}
 	data := struct{ Flags []shared.Flag }{flags}
-	componentTemplates.ExecuteTemplate(w, "wpt-env-flags.js", data)
+	if componentTemplates.ExecuteTemplate(w, "wpt-env-flags.js", data); err != nil {
```

Comment:
I'd personally prefer the consistency, but I'll leave it up to you.

Reply:
Actually, we can't use `RenderTemplate` here, as it's rendering a different "box" of templates (`componentTemplates` instead of `templates`). I'll leave this as is.